data/frameProcessor/src/CalibratorReset.cpp

```diff
@@ -7,11 +7,12 @@
 
 
 // it's probably a bad idea to put this here!
-static const int max_threads = 16;
+static const int max_threads = 6;
```

Comment:
Does reducing threads have an effect on the performance?

Reply:
yes and I chose 6 deliberately. Some work needs to be done finding the best value, but that I will do later.